controllers/disruption_controller.go

```diff
@@ -1202,7 +1202,7 @@ func (r *DisruptionReconciler) generateChaosPods(instance *chaosv1beta1.Disrupti
 		}
 
 		// get the ip ranges of cloud provider services
-		if instance.Spec.Network.Cloud != nil {
+		if instance.Spec.Network != nil && instance.Spec.Network.Cloud != nil {
```

Comment:
I thought this change was already on `main`? If it isn't, then `main` is broken right now?

Reply:
https://github.com/DataDog/chaos-controller/blob/e4a67cc099308b8f57d0f13941d801ed180de4b0/controllers/disruption_controller.go#L1205
Not sure why it shows this. But I struggled with merging 😅

Reply:
I think the GitHub view is misleading. It's the commit where I pulled from AWS onto my branch, the changes are from AWS here

Reply:
Huh, that's weird. Github /shrug